src/components/sortPetList.js

```diff
@@ -0,0 +1,23 @@
+
+function sortPetList(animalType, zipcode, sortParam) {
```

Comment:
This is almost identical to `fetchPetList`. You should be able to modify `fetchPetList` to optionally include sort params similar to what was done for type and location. Not a huge deal, but definitely an easy place to remove some duplicated code.